test/packages/EABase/include/Common/EABase/config/eacompiler.h

```diff
@@ -726,9 +726,9 @@
 			// Extern template is supported.
 		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__EDG_VERSION__) && (__EDG_VERSION__ >= 401)  // EDG 4.1+.
 			// supported.
-		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__APPLE__) && (EA_COMPILER_VERSION >= 401)
+		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__apple_build_version__) && (EA_COMPILER_VERSION >= 401)
```

Comment:
Yup.  Got that.  But we are already testing for both "__APPLE__ && __clang__" which would be equivalent.  Unless I'm missing something?  The __clang__ would already prevent using GCC on MacOSX, for example.